user_docs/en/userGuide.t2t

```diff
@@ -2755,6 +2755,7 @@ Following are the command line options for NVDA:
 | None | --no-sr-flag  | Don't change the global system screen reader flag |
 | None | --install | Installs NVDA (starting the newly installed copy) |
 | None | --install-silent | Silently installs NVDA (does not start the newly installed copy) |
+| None | --enable-start-on-logon=True|False | When installing, enable NVDA's start on the logon screen |

```

Comment:
I think we could clarify that the default behaviour for a fresh install is to start NVDA on the logon screen, probably best to put that in the start at windows logon section. Then we link to the `StartAtWindowsLogon` section here:
```suggestion
| None | --enable-start-on-logon=True|False | When installing, enable NVDA's [start on the logon screen #StartAtWindowsLogon] |
```